static/styles/cart.css

```diff
@@ -0,0 +1,77 @@
+body{
+    font-family: Dosis;
+    background-color: #dcdcdcde;
+}
+body>div:nth-child(1){
+    background-color: black;
+    color: white;
+    position: relative;
+    padding: 10px;
+}
+body>div:nth-child(1)>p{
+    margin: 0px;
+}
+body>div:nth-child(1)>button{
+    background-color: black;
+    position: absolute;
+    right:7px;
+    top:4px;
+}
+body>div{
+    background-color: white;
+    margin: 10px 0px;
+}
+.cartDetails{
+    display: flex;
+    /* padding:10px; */
+}
+.cartDetails>section{
+    padding:10px
+}
+.cartDetails>section:nth-child(1){
+    height:100px;
+}
+.cartDetails>section:nth-child(1)>img{
+    height: 100%;
+    object-fit: cover;
+}
+/* .itemdetails{
+    position: relative;
+} */
+.total{
+    position: fixed;
+    right:20px;
+    /* top:10px; */
+}
+.round{
+    border-radius: 50%;
+    margin: 10px;;
+}
+button{
+    background-color: #e91e63;
```

Comment:
styles for common elements like button span p etc must exist in separate file, say common.css